### ISSUE TITLE
all images on left please have auto align left

### ISSUE THREAD

**Issue by JacquelineMorck** [UnionDesign/keyin#5]
in postcard situation

<img width="1230" alt="screen shot 2016-06-01 at 11 28 59 am" src="https://cloud.githubusercontent.com/assets/18389551/15721049/13c1fd90-27ec-11e6-8217-e1e91f84222d.png">

same issue as on home page

more white space between image and text

<img width="1279" alt="screen shot 2016-06-01 at 11 18 13 am" src="https://cloud.githubusercontent.com/assets/18389551/15720767/ac290d6e-27ea-11e6-8cfb-c89d4aa7cbf2.png">

**Comment by UnionDesign** [comment]
I bumped up the column gutters site-wide to add a touch more white space in general to multi-column layouts and for the postcards specifically I bumped the gutters up a lot. So there'll be more white space now. Also made the images float left if the image is on the left and float right if the image is on the right. Note that the "decide" image was cropped with white space IN the image on both sides... just to keep it in proportion to the other drawing. But we could crop that out and if would float even further left (or right) adding that much more space between the image and text.
